### PR TITLE
Niv/cert 5866 check on target call

### PR DESCRIPTION
**Pull request by nivcertora**

### CODE REVIEW

File: RevertConditions/TargetCall/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+1. **Certora Run Command**
+    - Command:
+        ```bash
+        certoraRun TargetCall.conf
+        ```
+
+2. **Results Analysis**
+    - Analysis Link: [Certora Run Output](https://prover.certora.com/output/1512/4fe1e430860341aeb29dd19aa47c5e5e?anonymousKey=b49ee3115e09aad558bfd6f27ee974e2c4233e40)
+    - The analysis indicates that the `execute` function resulted in a revert due to the target contract not being a contract address: `tacRC!!53==0x0`.
```
Comment (shoham-certora) [inline RevertConditions/TargetCall/README.md:16]: Do we know why `Extcodesize` is not mentioned as the revert cause? It used to be.

File: RevertConditions/TargetCall/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Revert On Return Size Example
+
+## Overview
+This repository hosts a smart contract named `TargetCall`, featuring a function called `forward`. This function initiates the `execute` function of a specified target contract, set during the constructor invocation. The purpose of this example is to illustrate a revert condition based on the `TargetCall`. The compiler includes a check ensuring that the called target is a contract (verifies the code size). However, the prover can bypass this check by passing a regular address, resulting in a revert of the function.
```
Comment (shoham-certora) [inline RevertConditions/TargetCall/README.md:4]: Perhaps clarify some things:
- "regular address _(i.e. one without code)_"
- "the prover can _bypass this check_" -> "the Prover can _create a violation_"
- "The compiler _includes_ a check" -> "The compiler _inserts_ a check"

**Review by shoham-certora — APPROVED** [review]
Looks fine, note two comments I made.

File: RevertConditions/TargetCall/README.md (hunk 1 above)
Comment (nivcertora) [reply RevertConditions/TargetCall/README.md:16]: No clue

**Pull request merged**